Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM php:7.2-fpm
+
+# Install PHP extensions
+RUN docker-php-ext-install pdo_mysql && docker-php-ext-install calendar
+
+# Install Composer
+RUN curl -sS https://getcomposer.org/installer | php -- --install-dir=/usr/local/bin --filename=composer
+
+# Install Git
+RUN apt-get update && apt-get install -y git
+
+# Install Node.js
+RUN curl -sL https://deb.nodesource.com/setup_14.x | bash - && apt-get install -y nodejs
+
+# Install Yarn
+RUN npm install -g yarn
+
+WORKDIR /usr/share/nginx/budget
+COPY . .
```

Comment:
If you use ``.`` you want to have a ``.dockerignore`` file with at least ``.git`` in it.

Reply:
Git is used in the `composer.json`. After successfully installing the dependencies, it grabs the version number from Git and stores it in `version.txt`.